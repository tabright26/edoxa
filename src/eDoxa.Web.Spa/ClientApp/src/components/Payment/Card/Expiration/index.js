import CardExpiration from "./Expiration";

export default CardExpiration;
