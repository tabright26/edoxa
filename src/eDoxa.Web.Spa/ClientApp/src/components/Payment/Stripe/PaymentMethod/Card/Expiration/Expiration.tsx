import React, { Fragment } from "react";
import CardExpirationMonth from "./Month";
import CardExpirationYear from "./Year";

interface CardExpirationProps {
  month: Number;
  year: Number;
  className?: string;
}

const CardExpiration = ({ month, year, className }: CardExpirationProps) => (
  <Fragment>
    <CardExpirationMonth month={month} className={className} />
    <span className={className}>/</span>
    <CardExpirationYear year={year} className={className} />
  </Fragment>
);

export default CardExpiration;
