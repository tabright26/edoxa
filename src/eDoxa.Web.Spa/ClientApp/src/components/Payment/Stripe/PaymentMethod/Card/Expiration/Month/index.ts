import CardExpirationMonth from "./Month";

export default CardExpirationMonth;
