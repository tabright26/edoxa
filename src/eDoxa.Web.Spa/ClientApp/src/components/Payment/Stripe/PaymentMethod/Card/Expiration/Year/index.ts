import CardExpirationYear from "./Year";

export default CardExpirationYear;
