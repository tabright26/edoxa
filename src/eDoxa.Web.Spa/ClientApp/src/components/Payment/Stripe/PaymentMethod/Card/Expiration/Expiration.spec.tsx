import React from "react";
import renderer from "react-test-renderer";
import Expiration from "./Expiration";

it("renders without crashing", () => {
  const tree = renderer.create(<Expiration month={1} year={2030} />).toJSON();
  expect(tree).toMatchSnapshot();
});
