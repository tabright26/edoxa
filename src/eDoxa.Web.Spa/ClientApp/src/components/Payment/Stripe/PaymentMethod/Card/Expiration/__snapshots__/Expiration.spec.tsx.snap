// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
Array [
  <span>
    01
  </span>,
  <span>
    /
  </span>,
  <span>
    30
  </span>,
]
`;
